firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -178,4 +178,31 @@ class TrackingProtectionTests: BaseTestCase {
         mozWaitForElementToExist(app.staticTexts.elementContainingText("Firefox has not connected to this website."))
         XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
     }
+
+    // https://mozilla.testrail.io/index.php?/cases/view/2693741
+    func testLockIconCloseMenu() {
+        navigator.openURL("https://www.mozilla.org")
+        waitUntilPageLoad()
+        // iOS 15 displays a toast for the paste. The toast may cover areas to be
+        // tapped in the next step.
+        if #unavailable(iOS 16) {
+            sleep(2)
+        }
+        // Tap "Secure connection"
+        navigator.nowAt(BrowserTab)
+        navigator.goto(TrackingProtectionContextMenuDetails)
+        mozWaitForElementToExist(app.staticTexts["mozilla.org"])
+        mozWaitForElementToExist(app.staticTexts["Secure connection"])
```

Comment:
could we use just one element to be sure the dialog is there or not? I am afraid that having labels, if just a word changes, this will require updates... as long as the info sheet is shown or not, that would be enough... 
what do you think?

Reply:
i will do that